src/Http/Http.Extensions/src/RequestDelegateFactory.cs

```diff
@@ -587,6 +589,49 @@ private static Expression BindParameterFromValue(ParameterInfo parameter, Expres
             return argument;
         }
 
+        private static MethodCallExpression CreateTryParseCall(MethodInfo tryParseMethod, ParameterInfo parameter, Expression parsedValue)
```

Comment:
Where is this being called now? It feels weird to have to redo all these checks after the TryParse MethodInfo has already been found.

Reply:
The logic has been integrated into **`TryParseMethodCache`** precisely in the cache. We can remove this call 🙂 .